apps/web-portal-client/lib/components/Button.jsx

```diff
@@ -1,26 +1,29 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import {childrenPropTypes} from '../utils/childrenPropTypes'
 
 import './button.css'
 
 export function Button(props) {
-  const {label, handleClick, children} = props
+  const {label, onClick, children, addClasses} = props
```

Comment:
"addClasses" sounds like a verb but it is not. plz use something like "cssClass" or "className"